src/Microsoft.ComponentDetection.Detectors/pip/ISimplePyPiClient.cs

```diff
@@ -0,0 +1,310 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ComponentDetection.Common.Telemetry.Records;
+using Microsoft.ComponentDetection.Contracts;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Polly;
+
+[assembly: InternalsVisibleTo("Microsoft.ComponentDetection.Detectors.Tests")]
+
+namespace Microsoft.ComponentDetection.Detectors.Pip;
+
+public interface ISimplePyPiClient
+{
+    Task<Stream> FetchPackageFileStreamAsync(string name, string version, PythonProjectRelease release);
+
+    Task<SimplePypiProject> GetSimplePypiProjectAsync(PipDependencySpecification spec);
```

Comment:
Can you add some documentation to each of these methods using the xmldoc format? i.e.

```
/// <summary>
/// Some description
/// <summary>
/// <para, name="name">Some parameter description</param>
```

Copilot can be useful to write these quickly.

Reply:
Ah, just saw you have them on the implementation below. The standard in C# is to have the documentation on the interface, and then put `/// <inheritdoc />` on the implementation.